Brocfile.js

```diff
@@ -1,20 +1,16 @@
+/* jshint node: true */
 /* global require, module */
 
 var EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 
-var app = new EmberAddon();
+/*
+  This Brocfile specifes the options for the dummy test app of this
+  addon, located in `/tests/dummy`
+
+  This Brocfile does *not* influence how the addon or the app using it
+  behave. You most likely want to be modifying `./index.js` or app's Brocfile
```

Comment:
Ah, this was the default text from the ember-cli upgrade. According to the notes this is needed for the dummy app that runs the acceptance test. I'll try removing it and if there's no error I'll leave it out